content/ngf/overview/gateway-architecture.md

```diff
@@ -52,205 +37,141 @@ Key functionalities include:
 
 ### Data Plane: Autonomous Traffic Management
 
-Each NGINX data plane pod is provisioned as an independent Deployment containing an `nginx` container. This container runs both the `nginx` process and the [NGINX agent](https://github.com/nginx/agent), which is responsible for:
+Each NGINX data plane pod can be provisioned as an independent Deployment or DaemonSet containing an `nginx` container. This container runs both the `nginx` process and the [NGINX agent](https://github.com/nginx/agent), which is responsible for:
 
 - Applying configurations: The agent receives updates from the control plane and applies them to the NGINX instance.
 - Handling reloads: NGINX Agent handles configuration reconciliation and reloading NGINX, eliminating the need for shared volumes or Unix signals between the control plane and data plane pods.
 
-With this design, multiple NGINX data planes can be managed by a single control plane, enabling fine-grained, Gateway-specific control and isolation.
-
 ### Gateway Resource Management
 
-The architecture supports flexible operation and isolation across multiple Gateways:
+Users can have multiple gateways running side-by-side in the same cluster. This supports flexible operation and isolation across Gateways:
 
 - Concurrent Gateways: Multiple Gateway objects can run simultaneously within a single installation.
 - 1:1 resource mapping: Each Gateway resource corresponds uniquely to a dedicated data plane deployment, ensuring clear delineation of ownership and operational segregation.
 
-### Resilience and Fault Isolation
-
-One of the primary advantages of this architecture is enhanced operational resilience and fault isolation:
-
-#### Control Plane Resilience
-
-In the event of a control plane failure or downtime:
-- Existing data plane pods continue serving traffic using their last-valid cached configurations.
-- Updates to routes or Gateways are temporarily paused, but stable traffic delivery continues without degradation.
-- Recovery restores functionality, resynchronizing configuration updates seamlessly.
-
-#### Data Plane Resilience
-
-If a data plane pod encounters an outage or restarts:
-- Only routes tied to the specific linked Gateway object experience brief disruptions.
-- Configurations automatically resynchronize with the data plane upon pod restart, minimizing the scope of impact.
-- Other data plane pods remain unaffected and continue serving traffic normally.
-
-This split architecture ensures operational boundaries between the control plane and data plane, delivering improved scalability, security, and robustness while minimizing risks associated with failures in either component.
-
 ---
 
-## High-Level Example of NGINX Gateway Fabric in Action
+## High-Level Overview of NGINX Gateway Fabric in Action
 
 This figure depicts an example of NGINX Gateway Fabric exposing three web applications within a Kubernetes cluster to clients on the internet:
 
 ```mermaid
 graph LR
```

Comment:
couple notes on this diagram:

Let's change the NGF Pod name to NGF Control Plane Pod, and NGINX Pod to NGINX Data Plane Pod, to reinforce the concepts that we already introduced in this doc.

Also, the NGF Pod is outside the Kubernetes cluster. It should be inside it, in the nginx-gateway namespace.

(same goes for the table that follows the diagram)